src/main/kotlin/com/londogard/textgen/normalization/Normalization.kt

```diff
@@ -0,0 +1,8 @@
+package com.londogard.textgen.normalization
+
+interface Normalization {
+    val temperature: Double
+    fun normalize(probabilities: List<Pair<Int, Double>>): List<Pair<Int, Double>>
+    // fun normalize(probabilities: List<Pair<Int,Double>>): List<Pair<Int, Double>>
+    fun normalize(probabilities: DoubleArray): DoubleArray
```

Comment:
Maybe "stupid" to have these with the same name? I understand why they have the same name though. Just a thought

Reply:
Yes, will take a look at that certainly